pkg/agent/controller/agent.go

```diff
@@ -92,6 +105,30 @@ func (a *Controller) serviceExportToRemoteMcs(obj runtime.Object) runtime.Object
 	return mcs
 }
 
+func (a *Controller) serviceToRemoteMcs(obj runtime.Object) runtime.Object {
+	svc := obj.(*corev1.Service)
+	_, err := a.lhClientSet.LighthouseV2alpha1().ServiceExports(svc.Namespace).Get(svc.Name, metav1.GetOptions{})
+	if err != nil {
```

Comment:
Checking for not found error explicitly may be more useful.

Reply:
@tpantelis It would help to have flag in transform function that distinguishes between create/update/delete, kinda like how we had in ODL DCNs.